R/pkg/R/client.R

```diff
@@ -42,6 +42,19 @@ determineSparkSubmitBin <- function() {
   }
   sparkSubmitBinName
 }
+# R supports both file separator in the file path (Unix : / and Windows: \) irrespective of the operating system
+# but when passing file path to Java or script program, it has to be converted according to operating system
+
+determinefileSeperator <- function() {
+  if (.Platform$OS.type == "unix") {
```

Comment:
Is anything which is not Unix guaranteed to be Windows? It seems like Windows is actually the special case here, so can we check its platform directly rather than assuming "not unix" is Windows?